### PR TITLE
Added k_is_text_status_bar_white option to all status bar styling

### LINKED ISSUE #538: Change StatusBar text Color to black
For iOS 8, 9 and up is it possible to change the statusbar text item color to black?

Please advice.. the customization.h has k_is_text_status_bar_white, but doesnt seem to work with the new ios..

TASKS

- [X] [BUG] Uploader view shows status bar always black https://github.com/owncloud/ios/issues/676 @mRoRo (won't fix)
- [X] [BUG][iPad] Message "Updating file" is not displayed https://github.com/owncloud/ios/issues/677 @mRoRo
- [X] [BUG] Message "Updating file" is not correctly view with some branding options https://github.com/owncloud/ios/issues/678 @mRoRo [FIXED]

PR: https://github.com/owncloud/ios/pull/659

### PR DESCRIPTION
**Pull request by mRoRo**
This closes #538

### CODE REVIEW

File: Owncloud iOs Client/Files/Preview/DetailView/DetailViewController.m [hunk 1]
```diff
@@ -1865,21 +1873,34 @@ - (void) reloadFileInView:(NSNotification *)notification {
...
         NSString *notificationText = [NSString stringWithFormat:@"%@ %@", NSLocalizedString(@"updating", nil), [nameFileToUpdate stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding]];
         DLog(@"name: %@",notificationText);
         [_notification displayNotificationWithMessage:notificationText completion:nil];
     }
 }
 
+- (BOOL) isDownloadingImageOrFile {
 
+    if ((self.galleryView != nil && [self.galleryView isCurrentImageDownloading]) || (self.galleryView == nil && self.isDownloading)) {
+        return TRUE;
```
Comment (nasli) [inline Owncloud iOs Client/Files/Preview/DetailView/DetailViewController.m]: maybe set as YES and NO

File: Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m [hunk 2]
```diff
@@ -293,6 +307,13 @@ - (void) showTextInStatusBar {
 }
 
 
+- (BOOL) isDownloadingImageOrFile {
+
+    if ((self.galleryView != nil && [self.galleryView isCurrentImageDownloading]) || (self.galleryView == nil && self.isDownloading)) {
+        return TRUE;
```
Comment (nasli) [inline Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m]: same here

File: Owncloud iOs Client/Files/Preview/DetailView/DetailViewController.m (hunk 1 above)
Comment (mRoRo) [inline Owncloud iOs Client/Files/Preview/DetailView/DetailViewController.m]: 👍 done!

File: Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m [hunk 3]
```diff
@@ -257,15 +257,21 @@ - (void) putUpdateProgressInNavBar {
...
+          [_updatingCancelButton setImage:[UIImage imageNamed:@"cancel_download.png"] forState:UIControlStateNormal];
+    }
+    
     [_updatingFileView addSubview:_updatingFileProgressView];
     [_updatingFileView addSubview:_updatingCancelButton];
     
     [self.navigationItem setTitleView:_updatingFileView];
     
-    [self performSelector:@selector(showTextInStatusBar) withObject:nil afterDelay:0.8];
+    [self performSelector:@selector(showTextInStatusBar) withObject:nil afterDelay:0.0];
```
Comment (mRoRo) [inline Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m]: 👍 done!

File: Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m (hunk 2 above)
Comment (mRoRo) [inline Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m]: 👍 done!

**Pull request merged**